tools/map/build_junguozhi.py

```
@@ -0,0 +1,480 @@
+        for c in got:
+            c.pop('dist', None)
+        gap = (b['cities'] - len(got)) if b['cities'] else 0
+        extra = []
+        if gap > 0:
+            known = {c['name'] for c in got}
+            for _, w in sorted(b['residual']):
+                if w in known or len(extra) >= gap:
+                    continue
+                known.add(w)
+                extra.append(dict(name=w, note='', tags=[], lon=None, lat=None,
+                                  resolution='CANDIDATE_REGION'))
```

Ruff (0.16.1)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)